code/modules/mob/dead/dead.dm

```diff
@@ -52,7 +52,13 @@ INITIALIZE_IMMEDIATE(/mob/dead)
 	if(client.holder)
 		. += "Players Ready: [SSticker.totalPlayersReady]"
 		. += "Admins Ready: [SSticker.total_admins_ready] / [length(GLOB.admins)]"
-
+	//SKYRAT EDIT BEGIN: Removing Round Timer
+	var/round_time = world.time - SSticker.round_start_time
+	var/real_round_time = world.timeofday - SSticker.real_round_start_time
+	. += "Server Time: [time2text(world.timeofday, "YYYY-MM-DD hh:mm:ss")]"
+	. += "Round Timer: [round_time > MIDNIGHT_ROLLOVER ? "[round(round_time/MIDNIGHT_ROLLOVER)]:[worldtime2text()]" : worldtime2text()]"
+	. += "Actual Round Timer: [time2text(real_round_time, "hh:mm:ss", 0)]"
+	//SKYRAT EDIT END
```

Comment:
This entire proc is short-circuited and returns `..()` after the round has started. This code is never reached by observers or new_players when it matters. https://github.com/Skyrat-SS13/Skyrat-tg/blob/e543e0f092487f1fe32cbb63f3770dec3bf8876d/code/modules/mob/dead/dead.dm#L36-L41